src/stats/inc/LikelihoodBase.h

```diff
@@ -45,7 +46,7 @@ class GslMatrix;
  */
 
 template <class V = GslVector, class M = GslMatrix>
-class BaseGaussianLikelihood : public BaseScalarFunction<V, M> {
+class LikelihoodBase : public BaseScalarFunction<V, M> {
```

Comment:
We're operating under the assumption users wanting Gaussian likelihoods would have subclassed one of the existing Gaussian likelihoods to handle their covariance case rather than subclassing `BaseGaussianLikelihood` to *extend* QUESO's existing Gaussian likelihood capabilities.